Firestore/Example/Tests/Integration/API/FIRIndexingTests.mm

```diff
@@ -92,11 +96,114 @@ - (void)testBadIndexDoesNotCrashClient {
                     "}";
 
   [self.db setIndexConfigurationFromJSON:json
-                              completion:^(NSError* error) {
+                              completion:^(NSError *error) {
                                 XCTAssertNotNil(error);
                                 XCTAssertEqualObjects(error.domain, FIRFirestoreErrorDomain);
                                 XCTAssertEqual(error.code, FIRFirestoreErrorCodeInvalidArgument);
                               }];
 }
 
+/**
+ * After Auto Index Creation is enabled, through public API there is no way to see the indexes
+ * sitting inside SDK. So this test only checks the API of auto index creation.
+ */
+- (void)testAutoIndexCreationSetSuccessfully {
```

Comment:
Can we move these tests to swift?

Reply:
I will do the exact same tests for swift when public api